### ISSUE TITLE
Second I2C on Arduino Due

### ISSUE THREAD

**Issue by JPluess** [finitespace/BME280#107]
hi there i am using arduino due, and have it connected to the second I2C 
i would need to call Wire1.begin(); is there a simple solution in the library to select between sda0 scl0 and sda1 scl1 ?
best wishes

**Comment by coelner** [comment]
No, there is no simple solution. The library needs a change to support multiple buses.

**Comment by finitespace** [comment]
#123
